apps/vetrina/src/Vetrina/Main.purs

```diff
@@ -42,7 +38,7 @@ import Vetrina.Purchase.Completed as PurchaseCompleted
 
 foreign import sentryDsn_ :: Effect String
 
-type Props = { onClose :: Effect Unit }
+type Props = { onClose :: Effect Unit, products :: Array Product }
```

Comment:
Why this formatting?

Reply:
Well, first of all it makes sense that whoever includes vetrina, also somehow tells it which products to show. Also, we don't want to hard code the descriptions of the products to vetrina, so it makes sense to just take the full products in props.

Reply:
I just meant why the object's items are on the same line 😅

Reply:
Oh 🤠 It's such a short definition it fits nicely in one line

Reply:
But apparently it's common in the code, so I won't be bothering anymore 😃

Reply:
Yeah, I don't know... I mean
```
type Props =
  { onClose :: Effect Unit }
```
would look a bit funny (although I suspect we have this style in the code as well :D)

But you're right, if there's > 1 attributes, they should be formatted in separate lines:
```
type Props =
  { onClose :: Effect Unit
  , products :: Array Product
  }
```